src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -153,13 +153,13 @@ public string CalculateBlobPath(string item, string relativePath)
             if (Path.GetExtension(item) == ".nupkg")
             {
                 Tuple<string, NuGetVersion> nugetPackage = GetNugetPackageVersion(item);
-                blobPath = Path.Combine(Path.Combine(relativePath, Path.Combine(nugetPackage.Item1, nugetPackage.Item2.ToFullString())), Path.GetFileName(item));
+                blobPath = Path.Combine(Path.Combine(relativePath, Path.Combine(nugetPackage.Item1, nugetPackage.Item2.ToFullString())), Path.GetFileName(item)).ToLowerInvariant();
             }
             else
             {
-                blobPath = item.Substring(item.IndexOf(relativePath));
+                blobPath = Path.Combine(relativePath, Path.GetFileName(item));
             }
-            return blobPath.ToLowerInvariant().Replace("\\", "/");
```

Comment:
Moving toLower to just the nupkg section to preserve cases for files

Reply:
why does the path need to be tolower for the nuget files?